src/components/Identity/processIdSyncs.js

```diff
@@ -35,15 +38,15 @@ const setControlObject = obj => {
 
   Object.keys(obj).forEach(id => arr.push(`${id}-${obj[id]}`));
 
-  cookie.set(`${namespace}idSyncControl`, arr.join("_"), {
-    expires: 6 * 30 // 6 months
-  });
+  alloyCookie.set(ID_SYNC_CONTROL, arr.join("_"));
 };
 
-export default ({ destinations, config, logger }) => {
+export default ({ destinations, config, logger, cookie }) => {
+  alloyCookie = cookie;
+
   if (config.idSyncsEnabled) {
```

Comment:
Just a suggestion: I would `return` if this config is `false`, and take the code out of the `if` statement's block.